pkg/transport/message.go

```diff
@@ -54,7 +54,7 @@ type transporterImpl struct {
 
 func (impl *transporterImpl) Send(m msg.Message) error {
```

Comment:
Additionally, a TrySend function can be added, which returns a specific error when sendCh is full, instead of blocking. This function is more suitable for certain specific scenarios.

Reply:
Yes，Adding TrySend is indeed a way to make it clearer. I didn't read the code thoroughly and may not fully understand the this function.